src/managers/reader.ts

```diff
@@ -1,8 +1,67 @@
-import { Platform } from 'react-native';
+import {
+  Platform,
+  NativeEventEmitter,
+  type EmitterSubscription,
+} from 'react-native';
 import MobilePaymentsSdkReactNative from '../base_sdk';
+import type { ReaderChangedEvent, ReaderInfo } from '../models/objects';
 
-export const startPairing = (): Promise<void> => {
-  return MobilePaymentsSdkReactNative.startPairing();
+export const readerEventEmitter = new NativeEventEmitter(
+  MobilePaymentsSdkReactNative
+);
+export let readerChangesObserver: EmitterSubscription;
+
+export const getReaders = (): Promise<ReaderInfo[]> => {
+  return MobilePaymentsSdkReactNative.getReaders();
+};
+
+export const getReader = (id: string): Promise<ReaderInfo | null> => {
+  return MobilePaymentsSdkReactNative.getReader(id);
+};
+
+export const forget = (id: string): Promise<void> => {
+  return MobilePaymentsSdkReactNative.forget(id);
+};
+
+export const blink = (id: string): Promise<void> => {
+  return MobilePaymentsSdkReactNative.blink(id);
+};
+
+export const isPairingInProgress = (): Promise<boolean> => {
+  return MobilePaymentsSdkReactNative.isPairingInProgress();
+};
+
+const addReaderChangedCallback = (refId: String): Promise<String> => {
+  return MobilePaymentsSdkReactNative.addReaderChangedCallback(refId);
+};
+
+const removeReaderChangedCallback = (refId: String): Promise<void> => {
+  return MobilePaymentsSdkReactNative.removeReaderChangedCallback(refId);
+};
+
+export const setReaderChangedCallback = (
+  callback: (event: ReaderChangedEvent) => void
+): (() => void) => {
+  const refId = generateUUID();
+  addReaderChangedCallback(refId);
+  const subscription = readerEventEmitter.addListener(
+    //FIXME: use only ReaderChanged, the refId only to map thw listeners in native code
```

Comment:
Yeah, this.  I don't think we need a UUID refId (vs a fixed constant if you must, but I'm not sure you even need that) for the underlying `setReaderChangedCallback`.